docs/Deploy_FATE_Cluster_with_Admin_Role_in_Certain_Namespace.md

```diff
@@ -0,0 +1,138 @@
+# Deploy FATE Cluster with Admin Role in Certain Namespace
+
+## Background
+
+When deploying KubeFATE and FATE cluster with Kubernetes, users don't always have the cluster-admin privilege of the whole K8s cluster. To deploy KubeFATE and run FATE jobs, [admin]([admin](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles)) role provided by Kubernetes is necessary in certain namespace. And in this case, some configuration files need to be modified.
+
```

Comment:
Could you also add some description and the reference to "https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles" so that reader can further understand the different roles?